app/controllers/instrument_relays_controller.rb

```diff
@@ -0,0 +1,38 @@
+class InstrumentRelaysController < ApplicationController
+
+  admin_tab :all
+  before_action :authenticate_user!
+  before_action :init_product
+  before_action :manage
+
+  layout "two_column"
+
+  # GET /facilities/:facility_id/instrument/:instrument_id/relays
+  def index
+  end
+
+  # GET /facilities/:facility_id/instrument/:instrument_id/relays/:id/edit
+  def edit
+  end
+
+  def update
+  end
+
+  def new
+  end
+
+  private
+
+  def init_product
+    id_param = params.except(:facility_id).keys.detect { |k| k.end_with?("_id") }
```

Comment:
Since we are just using this for instruments, I think we  can do:
`@product = current_facility.instruments.find_by!(url_name: params[:instrument_id])`